spec/requests/exception_hunter/errors_request_spec.rb

```diff
@@ -1,6 +1,26 @@
 require 'rails_helper'
 
 describe 'Errors', type: :request do
+  describe 'show' do
+    let(:last_error) { ExceptionHunter::Error.last.id }
+
+    subject { get "/exception_hunter/errors/#{last_error}" }
+
+    before do
+      2.times do |i|
+        create(:error_group).tap do |error_group|
+          create_list(:error, i + 1, error_group: error_group)
+        end
+      end
+    end
```

Comment:
Why do we need to create so many things?

Reply:
we don't